scripts/lite/byron-testnet.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
```

Comment:
This script was renamed to indicate that this generates a RealPBFT (Byron) cluster